to.etc.domui/src/main/java/to/etc/domui/component/misc/MsgBox.java

```diff
@@ -287,6 +287,16 @@ public static void info(NodeBase dad, NodeContainer string) {
 		message(dad, Type.INFO, string);
 	}
 
+	/**
```

Comment:
Adding Javadoc is really good, but do not add those empty @param tags. We only add those (and the @result and @throws too) if they really need explanation. You can disable the automatic generation of those things in IntelliJ.